packages/core/src/editor/simplifiedBoxAPI/box-util-helpers/UtilPartHelpers.ts

```diff
@@ -68,7 +68,7 @@ export class UtilPartHelpers {
             BoxFactory.action(
                 element,
                 RoleProvider.property(element.freLanguageConcept(), propertyName, "new-list-item"),
-                `<+ ${propertyName}>`,
+                `${propertyName}`,
```

Comment:
I see that you would like to remove all extra character like "<", ">", "+" so the shown text is much cleaner. 
I your email example I do see the "+" again,  did you add that using CSS?